packages/vuetify/src/components/VTextarea/__tests__/VTextarea.spec.cy.tsx

```diff
@@ -16,7 +16,7 @@ describe('VTextarea', () => {
       .type('Eos q')
       .should(el => expect(el.outerHeight()).to.equal(56))
       .type('u')
-      .should(el => expect(el.outerHeight()).to.equal(82))
+      .should(el => expect(el.outerHeight()).to.equal(80))
```

Comment:
chips height decreased by 2px, which means small chip's height is consistent with old input line-height (24px).

So line-height can be rolled back

https://github.com/vuetifyjs/vuetify/pull/17175/files#r1181077790

https://github.com/vuetifyjs/vuetify/pull/17175/files#r1194469872

![image](https://github.com/vuetifyjs/vuetify/assets/5698884/1bf06910-2924-4e0a-a4d2-a5b980e2109e)